deployer/commands/generate/cryptnono_config/__init__.py

```diff
@@ -0,0 +1,34 @@
+import shutil
+import subprocess
+from pathlib import Path
+
+from deployer.cli_app import generate_app
+
+HERE = Path(__file__).parent
+REPO_ROOT_PATH = HERE.parent.parent.parent.parent
+
+
+@generate_app.command()
+def cryptnono_secret_config():
+    """
+    Update the secret blocklist for cryptnono
```

Comment:
I wanted to make this code and command not require pre-requisite knowledge on what this what about, but allow the help string (from the docstring) to convey that.

```suggestion
    Updates secret config for the cryptnono chart bundled with the support chart, specifically banned command strings and patterns as required by cryptnono's execwhacker detector if its enabled.
    
    The banned strings are generated using a secret script, and the deployer reads the secret config when deploying the support chart.
```

Reply:
This feels a little too repetitive for my personal style. Instead, once the PR is merged, I'll link to the docs from the code comments here - I put a lot of effort into those, and I think that's a better place for this kind of information.